fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1276,20 +1277,16 @@ ACTOR Future<Void> commitBatch(
 		ASSERT(p.second.isReady());
 	}
 
+	// After logging finishes, we report the commit version to master so that every other proxy can get the most
```

Comment:
It will be better if change the comment a bit:
"After logging finishes, we report the commit version to master  before update self->committedVersion so that ..."

Reply:
Since the ordering is quite sensitive and non-obvious, I feel it's better to specify the invariant like: "Maintain the invariant that master's committed version >= self->committedVersion by reporting commit version first before updating self->committedVersion. Otherwise, a client may get a commit version that the master is not aware of, and next GRV request may get a version less than self->committedVersion.".

Reply:
Done. Thanks for the suggestion!